typescript/nomad-deploy/src/core/CoreDeploy.ts

```diff
@@ -149,6 +151,7 @@ export class CoreDeploy extends Deploy<CoreContracts> {
     };
 
     for (var remote of remotes) {
+      if (!remote.contracts.replicas[local.chain.domain]) continue; // TODO: clean it. It prevents from connecting not connected networks
```

Comment:
remove TODO, i think this is fine

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/17/commits/01169d81534a0a5111b20db912c2f7cafd721852